### PR TITLE
Update scene list

### LINKED ISSUE #10: Updated scene list
I found this scene preset list that seems more up to date than the gist file -> https://github.com/Hypfer/hass-scene_presets/blob/master/custom_components/scene_presets/presets.json
It does seem to be in a different format so you would probably have to change your json parsing script 😅 

I would also filter out the `Misc Additions` category, these seem custom to that project.

### PR DESCRIPTION
**Pull request by Glennmen**
I created a separate pull request for the update scene list so that the convert script PR wouldn't block this one in case you have any change requests.

Closes #10 

Would also be nice to add the `hacktoberfest-accepted` label to this Pull Request 😇

### CODE REVIEW

File: ha-scenes.yaml [hunk 1]
```diff
@@ -547,44 +621,55 @@ variables:
...
       "Lily":{"c":[[0.5793,0.392],[0.5099,0.2918],[0.5705,0.272],[0.4703,0.3501],[0.5407,0.4008]],"b":0.796,"s":"Lush"},
+      "Lovebirds":{"c":[[0.5528,0.3806],[0.5787,0.393],[0.5149,0.4116],[0.5014,0.4153],[0.5202,0.4369]],"b":0.396,"s":"Romantic"},
       "Magneto":{"c":[[0.4215,0.4895],[0.1593,0.1341],[0.516,0.4401],[0.1575,0.2128],[0.171,0.3389]],"b":0.878,"s":"Futuristic"},
       "Majestic morning":{"c":[[0.3979,0.413],[0.5471,0.3588],[0.4741,0.43],[0.4659,0.3561],[0.1879,0.0644]],"b":0.569,"s":"Dreamy"},
-      "Malibu pink":{"c":[[0.5573, 0.3226],[0.3678, 0.1839],[0.1561, 0.1606],[0.4548, 0.2433],[0.154, 0.0799]],"b":0.81,"s":"Special"},
       "Memento":{"c":[[0.1554,0.0996],[0.1729,0.146],[0.4964,0.4542],[0.1824,0.0807],[0.1671,0.0853]],"b":0.4,"s":"Luxurious"},
       "Meriete":{"c":[[0.4636,0.4138],[0.1685,0.055],[0.2943,0.1842],[0.1547,0.1072],[0.2092,0.1108]],"b":0.796,"s":"Lush"},
-      "Miami":{"c":[[0.3678,0.1839],[0.154,0.0799],[0.5573,0.3226],[0.1561,0.1606],[0.4548,0.2433]],"b":0.796,"s":"Party vibes"},
+      "Miami":{"c":[[0.1621,0.3061],[0.5573,0.3409],[0.5335,0.3789],[0.2007,0.2178],[0.5371,0.2735]],"b":0.753,"s":"Race Day"},
```
Comment (nilsreiter) [inline ha-scenes.yaml:672]: It seems that Miami was part of "Party vibes", but now is in "Race Day" with different colors.

File: ha-scenes.yaml [hunk 2]
```diff
@@ -261,7 +312,6 @@ blueprint:
             - "Party vibes: Chinatown"
             - "Party vibes: Fairfax"
             - "Party vibes: Ibiza"
-            - "Party vibes: Malibu pink"
```
Comment (nilsreiter) [inline ha-scenes.yaml:264]: This seems to be missing from https://github.com/Hypfer/hass-scene_presets/blob/master/custom_components/scene_presets/presets.json

File: ha-scenes.yaml (hunk 1 above)
Comment (Glennmen) [reply ha-scenes.yaml:672]: @nilsreiter I see what has happened, there are 2 times Miami. Both in Race Day and in Party vibes.
Not sure how to solve this with current json implementation in the blueprint.

File: ha-scenes.yaml (hunk 2 above)
Comment (Glennmen) [reply ha-scenes.yaml:264]: @nilsreiter Not sure you want me to this, I could try to combine both json sources into 1 output.

File: ha-scenes.yaml (hunk 1 above)
Comment (nilsreiter) [reply ha-scenes.yaml:672]: Maybe it's time to ditch the existing format in the blueprint and just parse the other one ...

File: ha-scenes.yaml (hunk 2 above)
Comment (Hypfer) [reply ha-scenes.yaml:264]: @nilsreiter The integration allows users to add their own custom scene presets. See the documetation on that here: https://github.com/Hypfer/hass-scene_presets/blob/master/docs/Custom%20Presets.md

**Comment by Glennmen** [comment]
@nilsreiter I updated the json formated so that it now supports scenes with the same name.
Let me know what you think.

If approved I will update the script in PR #12

**Review by nilsreiter — APPROVED** [review]
Very nice, thanks!

**Pull request merged**